### PR TITLE
M2P-357 - Show alternative payment type in order grid

### PR DESCRIPTION
**Pull request by danac-gs**
# Description
Show payment type (such as Paypal) it in order grid as well, as done on order details page.

Fixes: https://boltpay.atlassian.net/browse/M2P-357

#changelog M2P-357 - Show alternative payment type in order grid

# Type of change

- [ ] Bug fix (change which fixes an issue)
- [x] New feature (change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?
Please validate that you have tested your change in at least one of the following areas:

- [ ] Successfully tested locally (or docker image)
- [x] Successfully tested on a staging or sandbox server
- [ ] Successfully tested on a merchant's staging server

# For PR Reviewer 
- [ ] Reviewed unit tests to make sure we are using real components rather than mocks as much as possible?
- [ ] For any major change (observer, new Bolt feature, core Magento interaction) we must add a feature switch, did you verify this?

# Checklist:

- [x] My code follows the style guidelines of this project.
- [x] I have performed a self-review of my own code.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] New and existing unit tests pass locally with my changes.
- [x] I have created or modified unit tests to sufficiently cover my changes.
- [x] I have added my Jira ticket link and provided a changelog message.

### CODE REVIEW

**Review by johnmccombs1 — DISMISSED** [review]

**Comment by danac-gs** [comment]
Additionally, invoice, credit memo and shipment grid tables are supported.

File: Helper/Order.php [hunk 1]
```diff
@@ -113,6 +113,7 @@ class Order extends AbstractHelper
         'paypal' => 'PayPal',
         'afterpay' => 'Afterpay',
         'affirm' => 'Affirm',
+	    'braintree' => 'Braintree'
```
Comment (dumega) [inline Helper/Order.php:116]: Please fix the indent

Comment (dumega) [inline Helper/Order.php:116]: Please fix the indent

**Review by dumega — COMMENTED** [review]
Please fix the indent

File: Helper/Order.php (hunk 1 above)
Comment (danac-gs) [reply Helper/Order.php]: done

**Review by johnmccombs1 — DISMISSED** [review]

**Review by dumega — APPROVED** [review]

File: Helper/Order.php [hunk 2]
```diff
@@ -113,6 +113,7 @@ class Order extends AbstractHelper
         'paypal' => 'PayPal',
         'afterpay' => 'Afterpay',
         'affirm' => 'Affirm',
+        'braintree' => 'Braintree'
     ];
```
Comment (vitaliyreznikov) [inline Helper/Order.php:117]: Looks like we need to add apple in this const as well after this PR https://github.com/BoltApp/bolt-magento2/pull/1043
@danac-gs Could you please check?

**Review by vitaliyreznikov — COMMENTED** [review]
@danac-gs Could you please add screenshots before and after?
For order grid only.

**Comment by danac-gs** [comment]
@vitaliyreznikov 
before
image
after
image

**Review by vitaliyreznikov — APPROVED** [review]

**Pull request merged**